src/screens/FeedScreen/RenderList.js

```diff
@@ -278,7 +278,7 @@ const styles = StyleSheet.create({
   cardContainer: {
     width: FULL_WIDTH,
     borderBottomWidth: 0,
-    borderBottomColor: COLORS.lightgrey,
+    borderBottomColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.lightgrey` to `COLORS.gray100` seems fine as it's a color adjustment. However, please ensure that this color change aligns with the overall design scheme and is consistent across all screens.
```diff
-    borderBottomColor: COLORS.lightgrey,
+    borderBottomColor: COLORS.gray100,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->